packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -366,3 +369,18 @@ export const selectCurrentTargetAnonymity = createSelector(
         return targetAnonymity;
     },
 );
+
+export const selectIsCoinjoinBlockedByTor = createSelector(
+    [selectSelectedAccountParams, selectTorState],
+    (selectSelectedAccountParams, { isTorEnabled }) => {
+        if (!selectSelectedAccountParams) {
+            return false;
+        }
+
+        if (selectSelectedAccountParams.symbol === 'regtest') {
```

Comment:
shoudnt it be `debug.coinjoinAllowNoTor` ?
we are using this condition in `RequestEnableTor` modal

Reply:
Yes, initially I thought we wanted to allow it only for `regtest` but since we have that setting property that allow you to use coinjoin without Tor it makes sense to use it here as well.

Done https://github.com/trezor/trezor-suite/pull/6796/commits/48dac020bdc1f089bb6c169c7d848c3a9b31f4b1